tests/core/test_actions.py

```diff
@@ -2732,3 +2732,76 @@ async def test_action_extract_slots_emits_necessary_slot_set_events(
         assert events[0].value == value_to_set
     else:
         assert len(events) == 0
+
+
+async def test_action_extract_slots_priority_of_slot_mappings():
+    slot_name = "location_slot"
+    entity_name = "location"
+    entity_value = "Berlin"
+
+    domain_yaml = textwrap.dedent(
+        f"""
+        version: "{LATEST_TRAINING_DATA_FORMAT_VERSION}"
+
+        intents:
+        - greet
+        - goodbye
+        - affirm
+        - deny
+        - mood_great
+        - mood_unhappy
+        - bot_challenge
+        - inform
```

Comment:
```suggestion
        intents:
        - inform
```

these tests can get hard to read so shortening the data portions to only the necessary